test/lib/usd/translators/testUsdExportPreviewShader.py

```diff
@@ -0,0 +1,69 @@
+#!/pxrpythonsubst
+
+#
+# Copyright 2020 Apple
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+import unittest
+
+import fixturesUtils
+from maya import cmds
+from maya import standalone
+from pxr import Usd
+
+
+def apply_preview_surface(node, shader=None):
+    """Apply a USD Preview Surface shader to the given node"""
+    shader = shader or cmds.shadingNode('usdPreviewSurface', asShader=True)
+    cmds.select(node)
+    cmds.hyperShade(assign=shader)
```

Comment:
Sadly, the `hypershade` command is not available in batch mode. This means that the shader must be setup the painful way.
I think the quickest way out of this would be to add opacity threshold checks in the existing `test\lib\usd\translators\testUsdExportImportRoundtripPreviewSurface.py` test.

Reply:
Ah good to know. We've been running tests in UI mode so it didn't come up. I'll move this over to the other test you mentioned.

Reply:
I'm not sure if you have privileges to change the Assignee, but this is the way to trigger a preflight for pull-requests. Results for your pull-request can be found in the checks tab - see https://github.com/Autodesk/maya-usd/pull/917/checks

Reply:
Okay switched over to being in the `testUsdExportImportRoundtripPreviewSurface`.
Also set it up so it imports the value as well.

Reply:
@kxl-adsk I do not have that permission it looks like.